src/nimibook/books.nim

```diff
@@ -51,18 +51,90 @@ proc dumpHook*(s: var string, v: string) =
     s.setLen(at)
 
 proc dump*(book: Book) =
+  var book = book
+  let rootFolder = getEnv("nimibook_rootfolder")
```

Comment:
in the end, didn't we decide to use `book.json` to pass the path?

Reply:
This i what I use to write to the path in the json. ``book.path_to_root = getEnv("...")`` and then we write book as Json.

Reply:
The only issues is ``useNimibook`` also uses rootFolder in the same process (other process have the value passed by --define) and needs to know the rootfolder

Reply:
ah ok. as in the [comment above](https://github.com/pietroppeter/nimibook/pull/19#pullrequestreview-674953517) (not in the review). I would actually still keep the nimble file mechanism. can we revert this part?